### PR TITLE
[feat] Add servlet for getting places photo

### PR DESCRIPTION
**Pull request by leonyeo**
Added servlet for getting photos from Places Photo api.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -121,12 +121,17 @@ function updateRestaurantList(restaurants) {
...
-    //       + photoReference + "&key=API_KEY_HERE";
-    //   resElement.appendChild(imgElement);
-    // }
+    if(photoReference) {
+      const url = new URL('/places-photo', window.location.origin),
+        params = {
+          photoRef: photoReference
+        }
+      Object.keys(params).forEach(key => url.searchParams.append(key, params[key]));
+
```
Comment (jieliangang) [inline src/main/webapp/script.js:130]: What happens if server return error?

Comment (leonyeo) [reply src/main/webapp/script.js:130]: I have modified it to display a blank img if server returns error

**Review by lolfuljames — COMMENTED** [review]
Code looks good! Could you deploy it on a demo site to let us see the changes?

**Comment by leonyeo** [comment]
Deployed to
https://lyeo-sps-summer20.appspot.com/

**Review by lolfuljames — APPROVED** [review]
LGTM

**Pull request merged**